theia-full-docker/Dockerfile

```diff
@@ -176,18 +183,47 @@ RUN apt-get update && apt-get -y install openjdk-11-jdk maven gradle
 
 # C/C++
 # public LLVM PPA, development version of LLVM
-ARG LLVM=13
-
-RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
-    echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main" > /etc/apt/sources.list.d/llvm.list && \
-    apt-get update && \
-    apt-get install -y \
-                       clang-tools-$LLVM \
-                       clangd-$LLVM \
-                       clang-tidy-$LLVM \
-                       gcc-multilib \
-                       g++-multilib \
-                       gdb && \
+# depending on architecture install form public LLVM PPA (Verison 13) or distirbution (Version 11)
+ARG LLVM=11
```

Comment:
We would probably still not want to touch this value, and only set it in the arm64 architecture only.

Reply:
Ok, I have changed the default to 13 and only downgrade it if you build the image on arm64.

Reply:
Awesome. If we go with one unified multi-arch image, I think we should proceed as per above when necessary. This way we maintain bleeding-edge features for the main arch, but permit other ones to evolve at their own pace and still work in the meantime, maybe with slightly reduced feature-set.